beater/beater.go

```diff
@@ -37,11 +36,13 @@ func (bt *beater) Run(b *beat.Beat) error {
 	}
 	defer pub.Stop()
 
+	go notifyListening(bt.config, pub.Send)
+
 	bt.server = newServer(bt.config, pub.Send)
-	err = run(bt.server, bt.config.SSL)
-	logp.Err(err.Error())
 
+	err = run(bt.server, bt.config.SSL)
 	if err == http.ErrServerClosed {
+		logp.Info("Listener stopped: %s", err.Error())
```

Comment:
We do send a document when the server is started, but we only log an information when it is stopped. Hence, having a document that the server was started in ES is not a relieable indicator for the server actually running. 

I think the original idea was to push a document and regulary update this document with the information that the server is running. When the last update of the document is too long ago this would indicate the server having stoped.

Reply:
Such additional features will be covered by monitoring: https://github.com/elastic/beats/issues/3422